core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -7736,8 +7736,15 @@ private void checkLiteral2(String expression, String expected) {
   }
   @Test void testSelectCountStar() {
     final String query = "select count(*) from \"product\"";
-    final String expected = "SELECT COUNT(*)\n"
-        + "FROM \"foodmart\".\"product\"";
+    final String expected = " ";
+    Sql sql = sql(query);
+    sql.ok(expected);
+  }
+//
+  @Test void testSInterval() {
```

Comment:
This test case is passing without your changes(in master) as well, Write the correct test case that should fail in master and should pass in your branch.